system-tests/tests/docker-compose.yml

```
@@ -0,0 +1,21 @@
+services:
+
+  # A nginx based HTTP server to serve DIDs.
+  did-server:
+    container_name: did-server
+    image: nginx
+    volumes:
+      - ./resources/webdid:/usr/share/nginx/html
+    ports:
+      - "80:80"
```

```suggestion
      - "8080:80"
```
expecting issues for some developers otherwise